index.js

```diff
@@ -25,7 +25,8 @@ if (!Element.prototype.matches) {
 AFRAME.registerComponent('teleport-controls', {
   schema: {
     type: {default: 'parabolic', oneOf: ['parabolic', 'line']},
-    button: {default: 'trackpad', oneOf: ['trackpad', 'trigger', 'grip', 'menu']},
+    aimEvent: {default: 'aim'},
```

Comment:
I'd default these to trackpaddown and trackpadup maybe?

Reply:
That's why it's WIP because I wanted to use `input-mapping` to provide a default mapping so most of the controllers will work out of the box instead of defaulting to a buttons that won't work for oculus for example, I'll default to a action itself

Reply:
I'd make teleport-controls out-of-the-box and agnostic towards mappings. So it should just work with buttons without configuration of mappings or emitting extra events. But if people use mappings, they can configure the events.